flashbax/vault/vault.py

```diff
@@ -235,11 +230,7 @@ def _get_base_spec(self, name: str) -> dict:
                 "base": f"{DRIVER}{self._base_path}",
                 "path": name,
             },
-            "metadata": {
-                "compressor": {
-                    **self._compression,
-                }
-            },
+            "metadata": {},
```

Comment:
I might be misunderstanding but should we not be returning self._metadata here as its the shared common metadata?